imageregistry/v1/types.go

```diff
@@ -49,10 +49,6 @@ type Config struct {
 type ImageRegistrySpec struct {
 	// operatorSpec allows operator specific configuration to be made.
 	operatorv1.OperatorSpec `json:",inline"`
-	// managementState indicates whether the registry instance represented
-	// by this config instance is under operator management or not.  Valid
-	// values are Managed, Unmanaged, and Removed.
-	ManagementState operatorv1.ManagementState `json:"managementState"`
```

Comment:
Why are we dropping this field?

Reply:
it's actually embedded in the OperatorSpec above.

Reply:
Oh right, this will technically break Go consumers then, but shouldn't break yaml/json consumers, I think we can live with that

Reply:
Do we need to keep the required management state though? Can we move the field to optional without that being considered breaking? It would mean some consumer may expect it to be set and suddenly it's not

Reply:
> Do we need to keep the required management state though? Can we move the field to optional without that being considered breaking? It would mean some consumer may expect it to be set and suddenly it's not

Given our options, the gain for the apply client outweighs the break.